### PR TITLE
Add create customer functionality

### PR DESCRIPTION
**Pull request by anthonybenjamin**
This pull adds a post function for creating new customers to the  custController @Ejesalva840

### CODE REVIEW

File: src/main/java/com/example/demo/custController/CustCon.java [hunk 1]
```diff
@@ -34,6 +37,10 @@ public String deleteCustomers(@PathVariable Long id) {
 	  }
 	
 	//create
+	@PostMapping("/customers")
+	public String createcustmoer(@RequestBody Customer newCust) {
```
Comment (Ejesalva840) [inline src/main/java/com/example/demo/custController/CustCon.java:41]: @anthonybenjamin, Hey, looks like there is a spelling error here, could you update this please?

File: src/main/java/com/example/demo/custController/CustCon.java [hunk 2]
```diff
@@ -14,7 +17,7 @@
 	CustomerRepository myclass;
 
 	//gets
-	@GetMapping("/customers")
+	@GetMapping("/")
```
Comment (Ejesalva840) [inline src/main/java/com/example/demo/custController/CustCon.java:20]: @anthonybenjamin, Could you also please add the customers mapping back as well?

**Review by Ejesalva840 — CHANGES_REQUESTED** [review]
Hey, left some feedback on a few things. Looks good otherwise!

File: src/main/java/com/example/demo/custController/CustCon.java (hunk 1 above)
Comment (anthonybenjamin) [reply src/main/java/com/example/demo/custController/CustCon.java]: @Ejesalva840 Thanks, got it done.

**Review by Ejesalva840 — APPROVED** [review]
Looks great!

**Pull request merged**